test/spec/ol/resolutionconstraint.test.js

```diff
@@ -96,54 +97,54 @@ describe('ol.resolutionconstraint', function() {
 
     beforeEach(function() {
       resolutionConstraint =
-          createSnapToPower(2, 1024, 10);
+          createSnapToPower(2, 1024, 1);
```

Comment:
Has the meaning of the 3rd argument changed? Or why is 1 used instead of 10 now?

Reply:
Yes, the third argument is now `minResolution` where it was before `maxZoomLevel`.
This was part of an effort to reduce mixing zoom levels and resolution values.